### PR TITLE
Replication

### PR DESCRIPTION
**Pull request by alainjobart**
To follow up on our email discussion on health check...

### CODE REVIEW

**Comment by alainjobart** [comment]
@enisoc @aaijazi 
(Ammar, I'll have to fix the droid guy upon import, to be simpler)

File: go/vt/health/health.go [hunk 1]
```diff
@@ -74,41 +66,38 @@ func NewAggregator() *Aggregator {
 // the first error will be returned.
 // It may return an empty map if no health condition is detected. Note
 // it will not return nil, but an empty map.
```
Comment (aaijazi) [inline go/vt/health/health.go]: function-level comments (what are they called in Go? godocs?) need updating - this no longer returns a map.

File: go/vt/health/health_test.go [hunk 2]
```diff
@@ -12,36 +12,32 @@ func TestReporters(t *testing.T) {
...
-	status, err := ag.Run(topo.TYPE_REPLICA, true)
+	delay, err := ag.Run(topo.TYPE_REPLICA, true)
 
 	if err != nil {
 		t.Error(err)
 	}
-	if want := map[string]string(map[string]string{"a": "value", "b": "value", "c": "value"}); !reflect.DeepEqual(status, want) {
-		t.Errorf("status=%#v, want %#v", status, want)
+	if delay != 10*time.Second {
+		t.Errorf("delay=%v, want 5s", delay)
```
Comment (aaijazi) [inline go/vt/health/health_test.go]: ...this error message should read "want 10s", yes?

**Comment by aaijazi** [comment]
Minor comment, but LGTM

**Pull request merged**